Better detail on custom Jupyter kernels

jmagosta opened the pull request:
Mention the "%azureml" error that occurs with the Iris tutorial if the notebook defaults to the "Python 3" kernel, and where in the interface one corrects it.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

jmagosta committed:
Better detail on custom Jupyter kernels 

Mention the "%azureml" error that occurs with the Iris tutorial if the notebook defaults to the "Python 3" kernel, and where in the interface one corrects it.

GitHubber17 committed:
Update how-to-use-jupyter-notebooks.md
